invenio_app_ils/circulation/api.py

```diff
@@ -40,7 +40,7 @@ def _validate_delivery(delivery):
 
 def _set_item_to_can_circulate(item_pid):
     """Change the item status to CAN_CIRCULATE."""
-    item = Item.get_record_by_pid(item_pid)
+    item = Item.get_record_by_pid(item_pid["value"])
```

Comment:
just thoughts for future: Item.get_record_by_pid should probably accept the `{pid: ... pid_type:...}` object and check if the record we try to get has a correct `pid_type`

Reply:
Sure, we can see how to improve this. But in this case, you know that you want exactly an `Item`, no? The thing is that it is a very specific case this thingy of `item_pid` that is a dict... if we had for all `***_pid` fields, then I agree, it would make a lot of sense!